### ISSUE TITLE
Unbunch option not working

### ISSUE THREAD

**Issue by KineticFox** [IncredibleHannes/TPF2-Timetables#23]
The unbunch isnt working. Trains ignore the time they should wait. 
But the arrival/departure is working as intended, trains are stopped and wait till departure time.

Tried aswell with a newly created line, same outcome...

**Comment by IncredibleHannes** [comment]
@KineticFox Can you provide more information?
Are you using it on a big map?

**Comment by KineticFox** [comment]
Depends, what is a big map ;) I have like round about 20-30 Train lines

**Comment by KineticFox** [comment]
I also tried differnt unbunch timings like 0:30 or 1:00 both arent working.

**Comment by IncredibleHannes** [comment]
Okay that narrows it down, can you try bigger unbunch times like 5 min, and tell me if they are working?

**Comment by IncredibleHannes** [comment]
might be related to https://github.com/IncredibleHannes/TPF2-Timetables/issues/17
If so then it should be fixed with the next update

**Comment by KineticFox** [comment]
Hm ok, timetable isnt working anymore. The UI wont open when clicking on the button.  I'll try tomorrow again and give feedback

**Comment by KineticFox** [comment]
UI still not working
stdout: 
```shell
C:\build\tpf2\src\Lib/UI/MemoryManager.h:28: class std::unique_ptr<class UI::ILayout,struct std::default_delete<class UI::ILayout> > __cdecl UI::MemoryManager::Remove<class UI::ILayout>(class UI::ILayout *): Assertion `it != m_data.end()' failed.

userdata: 00000215686FA6A8
```

**Comment by IncredibleHannes** [comment]
I already had someone else reporting similar issues. Somehow the UI state got corrupted, in this case only loading an old savegame or removing the mod, saving and adding the mod again helped.
I will investigate this further

**Comment by IncredibleHannes** [comment]
should be fixed with #38
